src/components/select-genre/select-genre.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import "./select-genre.css"
+
+
+export default function SelectGenre(props: any): React.ReactElement {
+    return <div className="selectGenre">
+        {props.genres.map((genre: string) => {
+            return <a
+                key={genre}
+                className={props.activeGenres.includes(genre) ? 'active' : ''}
+                onClick={() => props.setActiveGenres([genre])}
+            >
+                {genre}
+            </a>
+        })
+        }
+        <span style={{ width: '100%' }}></span>
```

Comment:
do we need this?

Reply:
I use it for styles:
![image](https://github.com/pratasenka/react-course/assets/99495416/03b59ac2-414b-4adb-8dad-6e2be0886341)